tools/packager/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "marathon"]
```

Comment:
I believe this can go as we are in the Marathon repo.

Reply:
As you mentioned we don't plan to push back upstream so just pulling this in is totally fine.